src/index.css

```diff
@@ -9,21 +9,256 @@ ul {
 }
 
 body {
-  background-color: #050518;
+  background-color: #fafafa;
   font-family: sans-serif;
   overflow: auto;
-  color: #fff;
+  color: #000;
 }
 
 .navbar {
   display: flex;
+  align-items: center;
   justify-content: space-between;
+  height: 5.938rem;
+  width: 100vw;
+  padding: 3rem 6rem;
+  border-bottom: 1px solid rgb(211, 205, 205);
+  background: rgb(255, 255, 255);
+}
+
+.Line {
+  width: 84rem;
+  height: 0.125rem;
+  margin: 0.93rem 6rem;
+  border: solid 1px #e8e8e8;
+}
+
+.footer {
+  margin: 0 6rem;
+}
```

Comment:
- [ ] Even the CR has required the change you didn't implement it, in the original design there are two inputs, please 🙏🏽 fix it. 👇🏽 

| Zeplin design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/232245270-4fe23cc3-5330-4b35-8d9c-b75a0f037d31.png) | ![image](https://user-images.githubusercontent.com/58642949/232245291-05444655-d48c-4c17-8293-e1521cca0c68.png) |

Reply:
thank you for the suggestion I missed this one 💯.